ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. -->
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4815743)